### PR TITLE
Fix typos

### LINKED ISSUE #11909: Little confused about the parenthesis
@ArisLittle commented on [Thu Apr 18 2019](https://github.com/dotnet/docs.zh-cn/issues/376)

Here is the open-parenthesis at "(Multicast“ and close-parenthesis at "algorithm)".
I guess that the close-parenthesis is placed at the wrong place. Is that so?

---
#### 文档详细信息

⚠ *请勿编辑此部分。 docs.microsoft.com ➟ GitHub 问题链接必须具有此部分。*

* ID: 134d3592-25e8-0650-b502-fa66ee0ed3c2
* Version Independent ID: b2b5c0f5-35d8-80da-2740-a099b75bbed0
* Content: [委托简介](https://docs.microsoft.com/zh-cn/dotnet/csharp/delegates-overview)
* Content Source: [docs/csharp/delegates-overview.md](https://github.com/dotnet/docs.zh-cn/blob/live/docs/csharp/delegates-overview.md)
* Product: **dotnet-csharp**
* GitHub Login: @BillWagner
* Microsoft Alias: **wiwagn**

### PR DESCRIPTION
**Pull request by pkulikov**
Fixes #11909

### CODE REVIEW

File: docs/csharp/delegates-overview.md [hunk 1]
```diff
@@ -39,16 +39,16 @@ any late binding algorithms. That enables developers to learn one
 concept, and use that same concept across many different software
 problems.
 
-Second, the team wanted to support both single and multi-cast method
+Second, the team wanted to support both single and multicast method
 calls. (Multicast delegates are delegates where multiple methods have
```
Comment (rpetrusha) [inline docs/csharp/delegates-overview.md]:
```suggestion
calls. (Multicast delegates are delegates that chain together multiple method calls. 
```

File: docs/csharp/delegates-overview.md [hunk 2]
```diff
@@ -39,16 +39,16 @@ any late binding algorithms. That enables developers to learn one
...
 been chained together. You'll see examples
```
Comment (rpetrusha) [inline docs/csharp/delegates-overview.md]:
```suggestion
You'll see examples
```

**Review by rpetrusha — DISMISSED** [review]
Your changes look good, @pkulikov. I've suggested an additional change unrelated to your PR -- I found the use of "where" in the sentence very annoyng.

**Comment by pkulikov** [comment]
@rpetrusha thanks for review and suggestions; I've applied them.

**Review by BillWagner — APPROVED** [review]
Thanks @pkulikov 

I'll review and :shipit: while @rpetrusha is out for a long weekend.

**Pull request merged**